src/pages/signup/index.js

```diff
@@ -47,82 +48,126 @@ const Signup = () => {
     handleSubmit,
     formState: { errors },
   } = useForm({ resolver: zodResolver(schema) });
-  const onSubmit = data => signup(data);
+  const [isSend, setIsSend] = useState(false);
+  const [toSend, setToSend] = useState({
+    to_name: '',
+    message: `Hello, thank you for signing up to Target MVD!`,
+    to: '',
+  });
+
+  const sendEmailValidation = data => {
+    setToSend(prevState => {
+      return {
+        ...prevState,
+        to_name: data.username,
+        to: data.email,
+      };
+    });
+  };
+
+  const onSubmit = data => {
+    sendEmailValidation(data);
+    signup(data);
+  };
+
   const resetErrors = useCallback(() => dispatch(api.util.resetApiState()), [dispatch]);
   const handleFocus = () => error && resetErrors();
-  useEffect(() => {
-    if (isSuccess) {
-      push(routesPaths.index);
+
+  const emailSendVerification = () => {
+    if (!isSend) {
+      emailjs
+        .send('service_5h9ht9l', 'template_se2l61q', toSend, 'XO-v6R9Eaw7t52iAM')
```

Comment:
Are these credentials? We should store this in an `.env` file so that they are not exposed in the code